sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala

```diff
@@ -956,3 +956,70 @@ class StreamingAggregationSuite extends StateStoreMetricsTest with Assertions {
     }
   }
 }
+
+class RocksDBStateStoreStreamingAggregationSuite
+  extends StreamingAggregationSuite with RocksDBStateStoreTest {
+  import testImplicits._
+
+  def snapshotVersionsPresent(dir: File): Seq[Long] = {
```

Comment:
Maybe better to have an helper object for RocksDB test to deduplicate this and below methods.

Reply:
Moved to a TestUtil trait.